src/renderer/dx/CustomTextRenderer.h

```diff
@@ -3,13 +3,17 @@
 
 #pragma once
 
+#include <d2d1_3.h>

```

Comment:
this probably belongs in a precomp

Reply:
Weirdly, it is in the precomp.h for all the projects I found that should be using it, but without it the build failed, but I didn't look too much into it. Definitely a FIXME.

Reply:
Correction: It is explicitly `#include`d in the `precomp.h` for `RendererDx`, but this file ends up included in a whole bunch of files. I couldn't find an appropriate file to add it, e.g. one with other `#include <d2d1*.h>` lines.

Reply:
Why does this end up included in other projects? Shouldn't it just be scoped inside `DxRenderer`?

Reply:
It's included in `DxRenderer.hpp` rather than forwarded, and that is included in `InteractivityWin32`'s `Window.cpp` and `TerminalControl`s `TermControl.{h,cpp}`. Seems like the right place to put it going by current style would be `DxRenderer.hpp` then, but better for those to have a pimpl pattern or get them onto `IRenderEngine`.